### PR TITLE
mixin: Make queries to find cluster and namespaces with running Mimir config configurable

### PR DESCRIPTION
**Pull request by pstibrany**
#### What this PR does

This PR extracts variable with PromQL queries to find clusters and namespaces with running Mimir in dashboards.

#### Checklist

- [na] Tests updated
- [na] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: operations/mimir-mixin/config.libsonnet [hunk 1]
```diff
@@ -51,6 +51,11 @@
     job_labels: [$._config.per_cluster_label, 'namespace', 'job'],
     cluster_labels: [$._config.per_cluster_label, 'namespace'],
 
+    // PromQL queries used to find clusters and namespaces with Mimir.
+    dropdown_job_query: 'cortex_build_info',  // Only used if singleBinary is true.
+    dropdown_cluster_query: 'cortex_build_info',
+    dropdown_namespace_query: 'cortex_build_info{%s=~"$cluster"}' % $._config.per_cluster_label,
```
Comment (dimitarvdimitrov) [inline operations/mimir-mixin/config.libsonnet:57]: nit: if these share the same prefix, can they be grouped into a `dropdowns` object within `_config`?

**Review by dimitarvdimitrov — APPROVED** [review]
LGTM. Not feeling strongly about the nitpick, merge either way

File: operations/mimir-mixin/config.libsonnet (hunk 1 above)
Comment (pstibrany) [reply operations/mimir-mixin/config.libsonnet]: Good idea, thanks.

File: operations/mimir-mixin/config.libsonnet [hunk 2]
```diff
@@ -51,6 +51,13 @@
     job_labels: [$._config.per_cluster_label, 'namespace', 'job'],
     cluster_labels: [$._config.per_cluster_label, 'namespace'],
 
+    // PromQL queries used to find clusters and namespaces with Mimir.
+    dropdown_queries: {
```
Comment (pracucci) [inline operations/mimir-mixin/config.libsonnet:55]: Have you considered calling it "dashboard_variables" to match the naming in Grafana?

**Review by pracucci — APPROVED** [review]

File: operations/mimir-mixin/config.libsonnet (hunk 2 above)
Comment (pstibrany) [reply operations/mimir-mixin/config.libsonnet:55]: These are not variables, but PromQL queries to select values for those dashboard variables. Would `dashboard_variables` fit?

Comment (pracucci) [reply operations/mimir-mixin/config.libsonnet:55]: To me it's still configuration for the dashboard variables, but I don't feel strong about it :) I already approved this PR :)

**Review by pracucci — APPROVED** [review]
It was good, it's still good, let's merge! 🚀 😉

**Pull request merged**